### DISCUSSION TITLE
Temporarily ignore assignees

### DISCUSSION THREAD

**Discussion by fabb** [renovatebot/renovate#23706] [Suggest an Idea]
### Tell us more.

We have multiple repos with different `reviewers` per-repo and we use `assigneesSampleSize: 1` for PR roulette. We also use a common base renovate config for all repos.

Sometimes a team member is out for a longer period of time (holiday, parental leave, sickness), and we'd like to remove this person from the renovate roulette temporarily. Currently we need to edit all repo's renovate configs where that team member is added as reviewer to do that.

It would be nice if there was a configuration option `ignoredAssignees` (or similar) that we can set in the common base renovate config to temporarily remove this team member from the PR roulette.

**Comment by rarkins** [comment]
Sounds OK. PRs welcome
